src/main/webapp/WEB-INF/views/driver/add.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create driver</title>
+</head>
+<body>
+<h1>Enter drivers name and license id</h1>
+<h1> Create Driver</h1>
+<form method="post" name="${pageContext.request.contextPath}/driver/add">
+    Name <input type="text" name="name">
+    Licesne id <input type="text" name="license_id">
+    <button type="submit">Save</button>
+</form>
```

Comment:
so did you check that after you create driver you can see him when you visit `/drivers` url?
for car and manufacturer check that after you add them, they are in db (run `select` query for that tables)

Reply:
Thank you, now all drivers display correctly, I run select query and found in db records that I created in web , hope now I did not miss something